api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -144,6 +145,12 @@ message HttpConnectionManager {
 
     // A list of custom tags with unique tag name to create tags for the active span.
     repeated type.tracing.v3.CustomTag custom_tags = 8;
+
+    // Configuration for an external tracing provider.
+    // If not specified, Envoy will fall back to using tracing provider configuration
+    // from the bootstrap config.
+    // [#not-implemented-hide:]
```

Comment:
FYI, you need to do some plumbing to ensure that the correct validator (derived from HCM's proto validator) is used when parsing the embedded tracer config.

Reply:
Do you mean `dynamicValidationVisitor()` vs `staticValidationVisitor()` ?

Or something else ?